docs/other/install-cvmfs-stratum1.md

```diff
@@ -0,0 +1,338 @@
+
+```
+[root@client ~] $ cvmfs_server add-replica -o root http://oasis.opensciencegrid.org:8000/cvmfs/oasis.opensciencegrid.org> /etc/cvmfs/keys/opensciencegrid.org/opensciencegrid.org.pub 
+[root@client ~] $ add_osg_repository http://cvmfs-stratum0.gridpp.rl.ac.uk:8000/cvmfs/config-egi.egi.eu
+```
+
+It's a good idea for other Stratum 1s to make their own scripts for adding repository replicas, because there's always two or three commands to run, and it's easy to forget the commands after the first one. The first command is generically this:
+
+```
+[root@client ~] $ cvmfs_server add-replica -o root http://cvmfs-stratum0.gridpp.rl.ac.uk:8000/cvmfs/config-egi.egi.eu> /etc/cvmfs/keys/egi.eu/egi.eu.pub
+```
+
+However, non-GOC OSG Stratum 1s (that is, at BNL and FNAL), for the sake of fulfilling an OSG security requirement, need to instead read from the OSG GOC machine with this as their first command:
+
+```
+[root@client ~] $ cvmfs_server add-replica -o root http://oasis-replica.opensciencegrid.org:8000/cvmfs/config-egi.egi.eu /etc/cvmfs/keys/egi.eu/egi.eu.pub:/etc/cvmfs/keys/opensciencegrid.org/opensciencegrid.org.pub
+```
+
+The second command for Stratum 1s that have the httpd configuration as described above in the [Configuring apache section](#Configuring_apache) is this:
+
+```
+[root@client ~] $ rm -f /etc/httpd/conf.d/cvmfs.config-egi.egi.eu.conf
+```
+
+Then the next command is this:
+
+```
+[root@client ~] $ cvmfs_server snapshot config-egi.egi.eu
+```
+
+With large repositories that can take a very long time, but with small repositories it should be very quick and not show any errors.
+
+## Verifying that the replica is being served
+
+Now to verify that the replication is working, do the following commands:
+
+```
+[user@client ~] $ wget -qdO- http://localhost:8000/cvmfs/config-egi.egi.eu/.cvmfspublished%7Ccat -v
+[user@client ~] $ wget -qdO- http://localhost:80/cvmfs/config-egi.egi.eu/.cvmfspublished%7Ccat -v
+```
+
+Both commands should show a short file including gibberish at the end which is the signature.
+
+It is a good idea to familiarize yourself with the log entries at `/var/log/httpd/access_log` and also, if you have installed frontier-squid, at `/var/log/squid/access.log`. Also, at least 15 minutes after the snapshot is finished, check the log `/var/log/cvmfs` to see that it tried to get an update and got no errors.
+
+# Setting up monitoring
+
+If you installed frontier-squid and frontier-awstats, there is a little more to do to configure monitoring.
+
+First, make sure that your firewall accepts UDP queries from the monitoring server at CERN. Details are on [the frontier-squid instructions](https://twiki.cern.ch/twiki/bin/view/Frontier/InstallSquid#Enabling_monitoring). Next, choose any random password and put it in `/etc/awstats/password-file`. Then tell Dave Dykstra the fully qualified domain name of your machine and the password you chose, and he'll set up the monitoring servers.
```

Comment:
Another old squid link

Reply:
Same here. This is a  CERN twiki link

Reply:
Ah, I saw twiki urls and didn't look closely enough, good point.